com.longbridgeapp.LongbridgePro.metainfo.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.longbridgeapp.LongbridgePro</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://support.longbridgeapp.com/topics/us-trade/user-agreement</project_license>
+  <name>Longbridge Pro</name>
+  <developer_name>LONG BRIDGE</developer_name>
+  <url type="homepage">https://longbridgeapp.com</url>
+  <summary>Trading platform.</summary>
+  <description>
+    <p>Longbridge Pro is  a professional trading platform tailored to your needs, with over 100 technical indicators and 12 multi-dimensional marking tools from Trading View, can elevate your strategy to a new level of value. </p>
+  </description>
+  <launchable type="desktop-id">com.longbridgeapp.LongbridgePro.desktop</launchable>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-info">moderate</content_attribute>
```

Comment:
Does the app involve monetary transactions? Then this rating does not seem correct. 

The description has trading, so I guess it does.

Reply:
Hi,

> Does the app involve monetary transactions? Then this rating does not seem correct.
> 
> The description has trading, so I guess it does.

Yes, it includes a stocks trading function and also a discussion platform.

I took a look at https://hughsie.github.io, I am not 100% about how can I rate it, so I removed this part about rating from my second try.

Thanks for the review!

Reply:
> I removed this part about rating from my second try

A rating must be present.

Reply:
Something like:

```
  <content_rating type="oars-1.1">
    <content_attribute id="social-info">moderate</content_attribute>
    <content_attribute id="money-purchasing">intense</content_attribute>
  </content_rating>
```
Intense means spending real money.